scripts/db_tools/import-project.ts

```diff
@@ -123,6 +123,8 @@ class Program {
     try {
       const originDB: Db = originClient.db();
       const destinationDB: Db = destinationClient.db();
+      console.log(`Origin realtime server connection: ${(originConn as any).state}`);
```

Comment:
Can you update the type definition in `src/RealtimeServer/typings/sharedb/lib/client.d.ts` instead of adding a type assertion here?